example/templates/server/sample-data/import.js

```diff
@@ -53,10 +53,6 @@ module.exports = function(app, cb) {
   });
 };
 
-if (require.main === module) {
-  // Run the import (server runs it automatically during boot)
-  var app = require('../server');
-  app.on('import done', function(err) {
-    process.exit(err ? 1 : 0);
-  });
-}
+if (require.main === module)
+  // The import runs automatically during the boot process.
+  require('../server');
```

Comment:
Yes, as long as `err` is returned in the callback (we `throw` by default in https://github.com/strongloop/loopback-workspace/commit/223758f6cb4051e4791c52ce6f8f2f88a4200f2d#diff-383e9ce062435fb34a4326cdf175c907R16 causing an `unhandledException` event causing Node to return 1 for the exit code). I did a smoke test by changing the name of the JSON file in `import.js` deliberately causing a missing file type error then checking the exit status in bash via `echo $?` which returned 1.